vendor/github.com/HewlettPackard/oneview-golang/ov/profiles.go

```diff
@@ -62,7 +62,7 @@ type BiosSettings struct {
 // BiosOption - bios options
 type BiosOption struct {
 	ConsistencyState   string         `json:"consistencyState,omitempty"`   //Consistency state of the BIOS component
-	ManageBios         bool           `json:"manageBios"`                   // "manageBios": false,
+	ManageBios         *bool          `json:"manageBios"`                   // "manageBios": false,
 	OverriddenSettings []BiosSettings `json:"overriddenSettings,omitempty"` // "overriddenSettings": []
 	ReapplyState       string         `json:"reapplyState,omitempty"`       //Current reapply state of the BIOS component.
 }
```

Comment:
If these changes are to be included , the go.mod files should reflect the newer commit for oneview-golang